### PR TITLE
Bump blockifier version

### LINKED ISSUE #1014: Research bumping blockifier
The task should include checking if it's technically possible,
and also as a follow up (if it is), checking which logic has been changed in the span of latest versions and create appropriate task of testing and updating our copied code.

If it's not possible now, please research the extent of the changes and leave the task to be done for 2.3.0 update (which should be pretty soon)

### PR DESCRIPTION
**Pull request by Arcticae**
Closes #1014

## Introduced changes

-

## Checklist

- [x] Linked relevant issue
- [x] Updated relevant documentation
- [x] Added relevant tests
- [x] Performed self-review of the code
- [x] Added changes to `CHANGELOG.md`

### CODE REVIEW

**Review by piotmag769 — APPROVED** [review]

File: crates/cheatnet/src/constants.rs [hunk 1]
```diff
@@ -95,26 +96,55 @@ pub fn build_block_context(block_info: CheatnetBlockInfo) -> BlockContext {
...
         block_timestamp: block_info.timestamp,
         sequencer_address: block_info.sequencer_address,
-        fee_token_address: ContractAddress(patricia_key!(TEST_ERC20_CONTRACT_ADDRESS)),
-        deprecated_fee_token_address: ContractAddress(patricia_key!(TEST_ERC20_CONTRACT_ADDRESS)),
         vm_resource_fee_cost,
-        gas_price: 100 * u128::pow(10, 9),
         invoke_tx_max_n_steps: 1_000_000,
         validate_max_n_steps: 1_000_000,
         max_recursion_depth: 50,
+        // TODO: Do 2 separate ones
```
Comment (war-in) [inline crates/cheatnet/src/constants.rs:103]: Do we have an issue for this TODO?

File: crates/forge/tests/e2e/io_operations.rs [hunk 2]
```diff
@@ -14,125 +14,110 @@ fn file_reading() {
...
+
+
+Collected 23 test(s) from file_reading package
+Running 9 test(s) from src/
+[FAIL] file_reading::tests::negative_number
+
+Failure data:
+    Got an exception while executing a hint: Hint Error: Failed to parse data/negative_number.txt file
+
+[FAIL] file_reading::tests::too_large_number
```
Comment (war-in) [inline crates/forge/tests/e2e/io_operations.rs:28]: Don't we want to format this as before?

**Review by war-in — APPROVED** [review]

File: crates/forge/tests/e2e/io_operations.rs (hunk 2 above)
Comment (Arcticae) [reply crates/forge/tests/e2e/io_operations.rs:28]: If i format this like it was formatted before it fails to match. This test is bullshit anyways, matching the output 1:1 is ridiculous if anyone asked me...

File: crates/cheatnet/src/constants.rs (hunk 1 above)
Comment (Arcticae) [reply crates/cheatnet/src/constants.rs]: I don't think it even matters. Removed.

**Pull request merged**